data/mods/blazemod/vehicleparts/blob_turret.json

```diff
@@ -0,0 +1,195 @@
+[
```

Comment:
Another modernisation was performed here, after separating out the blob turrets. The `blob_turret_mount` was expanded to include installation and removal times, a `folded_volume` that wasn't 1, and `damage_reduction` like the vanilla turret mount. After that, I made a `blob_turret` template part that contains basic fields that all turrets would have, and has helped trim down most of the entries of this type- there's a couple of weird weapons with their own durabilities and whatnot, and that has been retained.

Need feedback on the `folded_volume`, it was mostly a pull-over from vanilla and I'm not sure if I should have changed it from the value of 1- I could probably do balancing for that in a different PR.